Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -1204,6 +1204,10 @@ theorem normAtAllPlaces_mixedSpaceOfRealSpace {x : realSpace K} (hx : ∀ w, 0 
   ext
   rw [normAtAllPlaces_apply, normAtPlace_mixedSpaceOfRealSpace (hx _)]
 
+theorem normAtAllPlaces_mixedEmbedding (x : K) (w : InfinitePlace K) :
```

Comment:
This can be `simp`.

Reply:
Actually, that makes the simp linter complain...